service-api/config/autoload/local.php.dist

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * Local Configuration Override
+ *
+ * This configuration override file is for overriding environment-specific and
+ * security-sensitive configuration information. Copy this file without the
+ * .dist extension at the end and populate values as needed.
+ *
+ * NOTE: This file is ignored from Git by default with the .gitignore included
```

Comment:
Given we use containers and configure by environment you can get rid of this file (and it's non-dist version). The idea being your config differences for development should be made at the docker compose level, not in php code files. We tend to follow https://12factor.net/ where we can.